client/modules/IDE/components/PreviewFrame.jsx

```diff
@@ -333,7 +333,7 @@ class PreviewFrame extends React.Component {
     return (
       <iframe
         id="canvas_frame"
-        className="preview-frame"
+        className={this.props.fullView ? 'fullview-preview-frame' : 'preview-frame'}
```

Comment:
The pattern I've used in the web editor relies upon the [classnames](https://github.com/JedWatson/classnames) npm package, as well as [BEM](http://getbem.com/). Therefore, the code should look more like:
```js
// somewhere in render, above the return statement
let iframeClass = classNames({
  'preview-frame': true,
  'preview-frame--full-view': this.props.fullView
});

// in the return statement
<iframe
  ...
  className={iframeClass}
  ...
```
and `preview-frame--full-view` given the appropriate CSS.